ci-operator/step-registry/crio-test/e2e/test/crio-test-e2e-test-commands.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+echo "entering setup!!!!"
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/packet-conf.sh"
+
+echo "${SSHOPTS[@]}"
+echo "${IP}"
+echo "${CLUSTER_PROFILE_DIR}/packet-ssh-key"
+cat "${CLUSTER_PROFILE_DIR}/packet-ssh-key" 
```

Comment:
need to remove the cat here

Reply:
probably shouldn't have that there at all lol